packages/dynamite/dynamite_end_to_end_test/lib/types.openapi.json

```diff
@@ -55,6 +55,104 @@
             "Object": {
                 "type": "object"
             },
+            "AdditionalProperties": {
```

Comment:
Please add some tests that make use of this code, especially the nested variant.